README.md

```diff
@@ -1 +1,65 @@
-# BloomDiscordBot
\ No newline at end of file
+# BloomDiscordBot
+
+# Help:
+
+You can type ```$bot_help``` to get details about what commands can be used, along with a brief description of them
+
+# Events:
+
+When a person creates an event within Blooms Discord server, there will initially be a one hour delay from the time of creation to when the details of the event are posted. This will allow the person sometime to make changes to the details, timing and so on.
+
+There is a daily scheduled task that will identify events starting within the next 24 hours. If any are detected everyone is informed of the events details in a Discord message.
+
+Events can also be deleted through a command, detailed below.
+
+# Commands:
+
+The following commands can be used to manage, and list events.
+All commands start with: $
+
+**List Events:**
+
+```
+$list_events
+```
+
+**Example:**
+
+```
+$list_events
+```
+
+**Response:**
+
+```
+🗓️ All Events🗓️ 
+
+🌺 Scrutinizer Game night🌺 
+event_id: 1180231420476670043
+Description: 
+
+🌺 Modular Mesh Architecture🌺 
+event_id: 1184322631692980325
+Description:
+```
+
+**Delete Events:**
+
+```
+$delet_eevent <event_id>
+```
+
+**Example:**
+
+```
+$delete_event 1179242504395165748
+```
+
+**Response:**
+
+```
+Event with ID 1179242504395165748 has been deleted.
+```
```

Comment:
Is this still accurate?
if so how are people getting the event id?
if they have to go into the events well ... :)

Reply:
> Is this still accurate? if so how are people getting the event id? if they have to go into the events well ... :)

There is a bot command ``list_events`` which provides this detail. 

``` 
@bot.command(name='list_events')
async def listevents(ctx):
    guild = ctx.guild
    event_list = await check_upcoming_events(guild)

    # Extracting event information
    event_info_list = [(event.name, event.id, event.description) for event in event_list]

    # Formatting the information
    formatted_events = [
        f"🌺 **{name}**🌺 \n**event_id: **{event_id}\n**Description:** {description}"
        for name, event_id, description in event_info_list
    ]
    formatted_string = "\n\n".join(formatted_events)

    await ctx.send(f"🗓️ **All Events**🗓️ \n\n{formatted_string}")

```